tools/buildFactory.ts

```diff
@@ -174,73 +202,166 @@ export async function gulpUmdMinFactory(input: string, output: string) {
 }
 
 export function gulpIsolateFactory(
```

Comment:
If I don't get it wrong, this function will do someting to change `export['default']` to `exports.xxx = xxx`.

But I think it's better to change `src/xxx/xxx.ts` to `export {}`.

Reply:
I fixed the defect in the treeshaking by replacing index.ts in every time build.

Therefore, we do not convert export default to named export

Reply:
I mean change

```typescript
export class OrderedSetIterator {};

export default OrderedSet;
```

to

```typescript
class OrderedSetIterator {};

export type { OrderedSetIterator };

export { OrderedSet };
```

in `src/TreeContainer/OrderedSet.ts` directly.

Reply:
Does rollup-plugin-ts work fine in this case?